pkg/handlers/generic/lifecycle/utils/utils.go

```diff
@@ -219,3 +223,19 @@ func CreateConfigMapForCRS(configMapName, configMapNamespace string,
 	cm.Data[defaultCRSConfigMapKey] = string(utilyaml.JoinYaml(l...))
 	return cm, nil
 }
+
+func SetTLSConfigForHelmChartProxyIfNeeded(hcp *caaphv1.HelmChartProxy) {
+	// this is set as an environment variable from the downward API on deployment
+	deploymentNS := os.Getenv("POD_NAMESPACE")
```

Comment:
Is this env always set by some Kubernetes controller or something thats needs to be in the Pod spec?
(I couldn't find it in https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/)

Reply:
Its being set by the downward API already by the helm chart 

https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/9498fa43f0d0fcaa567dc2b776ca409a593f52d5/charts/cluster-api-runtime-extensions-nutanix/templates/deployment.yaml#L56

Reply:
Ahh thats an existing env, got it 👍